lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -228,9 +228,7 @@ export async function initRepo({
   }
 }
 
-export async function getRepoForceRebase(): Promise<boolean> {
-  logger.debug(`getRepoForceRebase()`);
-
+export async function getBranchForceRebase(): Promise<boolean> {
```

Comment:
```suggestion
export async function getBranchForceRebase(_branchName: string): Promise<boolean> {
```
add parameter for later usage?